src/com/palmergames/bukkit/towny/object/metadata/IntegerDataField.java

```diff
@@ -6,6 +6,10 @@
     public IntegerDataField(String key) {
         super(key, CustomDataFieldType.IntegerField);
     }
+
+	public IntegerDataField(String key, Integer value, String label) {
+		super(key, CustomDataFieldType.IntegerField, value, label);
+	}
```

Comment:
The label field doesn't need to be in any constructor its an optional field

Reply:
The main reason is so that it can be defined quickly when setting up the field in one line rather than requiring two lines.

Reply:
Ok I didn't see that you kept the older constructor as well, that's fine